Sources/MapLibreSwiftUI/MapView.swift

```diff
@@ -18,7 +18,7 @@ public struct MapView<T: MapViewHostViewController>: UIViewControllerRepresentab
     var onStyleLoaded: ((MLNStyle) -> Void)?
     var onViewPortChanged: ((MapViewPort) -> Void)?
 
-    public var mapViewContentInset: UIEdgeInsets = .zero
+    var mapViewContentInset: UIEdgeInsets? = .zero
```

Comment:
Two point of note here:

1. I apologise if I've forgotten a reason that this was supposed to be public, but I don't see an obvious reason so removed that modifier.
2. I made the insets optional because I think that it's quite possible that someone (maybe @Patrick-Kladek and @hactar?) actually DO want the default behavior! (More in a future comment at the use site.)